src/DataViews/CMakeLists.txt

```diff
@@ -32,6 +33,7 @@ target_sources(DataViews PUBLIC
         include/DataViews/PresetLoadState.h
         include/DataViews/SamplingReportDataView.h        
         include/DataViews/SamplingReportInterface.h
+        include/DataViews/ScopeDataView.h
```

Comment:
As all the DataView classes are tested. I think we should continue with this practice (if possible) and add test here.
Note: There is the `TYPED_TEST` feature in gtest.

Reply:
The abstract class introduces three protected methods (they are implementation details and shouldn't be tested, right?), while the two public methods are tested in the tests corresponding to the subclasses. It seems to be much easier to test all the context menu actions together (this is how it's done now). 

So, overall, I suggest to see this abstract class as an implementation detail.